File: Backend/src/main/java/Queue/management/system/example/management/system/beans/UserDetails.java
package Queue.management.system.example.management.system.beans;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder

public class UserDetails {
    private int userId;
    private String email;
    private String password;
    private String userName;
    private UserType userType;


}

File: Frontend/frontend/src/Models/UserDetails.ts
export class UserDetails {
    public userId: number;
    public email: string;
    public password: string;
    public userName: string;
    public userType: string;

   
    constructor(userId: number, email: string, password: string, userName: string, userType:string) {
        this.userId = userId;
        this.email = email;
        this.password = password;
        this.userName = userName;
        this.userType = userType;
    }
}